_tutorials/page10.md

```diff
@@ -104,14 +104,14 @@ Also, this should be the folder structure:
 
 ```
 - scanapi (root directory)
-  - .env
-  - scanapi-report.html
-  - scanapi.conf
-  - scanapi.yaml
-  - snippets.yaml
+|── .env
+|── scanapi-report.html
+|── scanapi.conf
+|── scanapi.yaml
+|___  snippets.yaml
```

Comment:
Small typo, the last one, `snippets.yaml`, is indented here

Reply:
@maiakovsky I did it on purpose, to make it aligned in the rendered version

![image](https://user-images.githubusercontent.com/2728804/121891696-2c297b00-ccf2-11eb-9e13-1249e076c0a5.png)

I tried to add directly the tree output, but look how it gets:

![image](https://user-images.githubusercontent.com/2728804/121891864-6135cd80-ccf2-11eb-854b-59a18b233065.png)